src/lem/circuit.rs

```diff
@@ -687,6 +687,8 @@ fn synthesize_block<F: LurkField, CS: ConstraintSystem<F>, C: Coprocessor<F>>(
                         bail!("Incompatible output length for coprocessor {sym}")
                     }
                     for ((i, var), ptr) in out.iter().enumerate().zip(synthesize_output) {
+                        // TODO COMPARE COPROC RESULT CIRCUIT
+
```

Comment:
Why is this comment here? That check happens precisely in the next few lines:
```rust
if ptr.tag().get_value() != Some(z_ptr.tag_field())
    || ptr.hash().get_value() != Some(*z_ptr.value())
{
    bail!("Mismatch between evaluate and synthesize outputs for coprocessor {sym} (pointer {i})")
}
```

Reply:
That's a left over from my debugging, my bad :disappointed:  will check thoroughly that I do not left such problem elsewhere